src/features/locators/LocatorsPage.tsx

```diff
@@ -65,6 +65,8 @@ export const LocatorsPage = () => {
   const containerRef = useRef(null);
   useNotifications(containerRef?.current);
 
+  const presentLocators = useSelector((state: RootState) => state.locators.present.entities);
```

Comment:
you should use `selectPresentLocatorsByPO` selector here. 
also, for shorter notation you can write 
```const isNoPageLocators = isEmpty(useSelector(selectPresentLocatorsByPO))```